src/ModelParser/JMSParser.php

```diff
@@ -145,7 +149,8 @@ private function parseClass(\ReflectionClass $reflClass, RawClassMetadata $class
         } catch (AnnotationException $e) {
             throw ParseException::classError($reflClass->getName(), $e);
         }
-        foreach ($attributes as $attribute) {
+        foreach ($attributes as $key => $values) {
```

Comment:
you seem not to use the key

```suggestion
        foreach ($attributes as $values) {
```

Reply:
Good catch. I removed and amended it to the original commit 25ceca84dd72508f8058455be20b63ccdf18e213